### PR TITLE
Updated comment_controller.rb of the issue Rubocop - Use string interpolation (comment_controller)

### LINKED ISSUE #11430: Rubocop - Use string interpolation (`comment_controller`)
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from underrepresented groups in free and open-source software!

We know that the process of creating a pull request is one of the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new** and looking through our general [help wanted](https://github.com/publiclab/plots2/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Claim the issue by commenting. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!
<details>
  <summary>💡 <i>Learn how to claim</i> 🙋</summary>
  <h3>Claiming an issue</h3>
  <p>Unless the issue is marked as reserved for someone, you can just say "I'd like to try this!" and then you've claimed it - no need to wait for someone to assign it to you. Just be sure you link your pull request (PR) to this issue so we can see where your solution is.</p>
  <p>And open one early if possible - even before you've completed it with additional commits - and others can help you figure out any issues you may face.</p>
</details>

- [ ] 📝 **Update** the file [app/controllers/comment_controller.rb](https://github.com/publiclab/plots2/blob/main/app/controllers/comment_controller.rb) in the `plots2` repository (press the little pen Icon) and edit the line as shown below.

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
@@ -45,9 +45,9 @@ class CommentController < ApplicationController
               "<a href='/subscribe/tag/#{tagname}'>#{tagname}</a>"
             end
             tagnames = tagnames.join(', ')
-            tagnames = " Click to subscribe to updates on these tags or topics: " + tagnames unless tagnames.empty?
+            tagnames = "Click to subscribe to updates on these tags or topics: #{tagnames}" unless tagnames.empty?
             flash[:notice] = "Comment posted.#{tagnames}"
-            redirect_to @node.path + '#last' # to last comment
+            redirect_to "#{@node.path}#last" # to last comment
           end
         end
       end
@@ -94,7 +94,7 @@ class CommentController < ApplicationController
       @comment.comment = params[:body]
       if @comment.save
         flash[:notice] = 'Comment updated.'
-        redirect_to @path + '?_=' + Time.now.to_i.to_s
+        redirect_to "#{@path}?_=#{Time.now.to_i}"
       else
         flash[:error] = 'The comment could not be updated.'
         redirect_to @path
@@ -127,7 +127,7 @@ class CommentController < ApplicationController
                 render json: { comment_count: @comment_count }
               else
                 flash[:notice] = 'Comment deleted.'
-                redirect_to '/' + @node.path
+                redirect_to "/#{@node.path}"
               end
             end
           end
@@ -197,7 +197,6 @@ class CommentController < ApplicationController

       if @comment.destroy
         render json: { success: true }
-        return
       else
         flash[:error] = 'The comment could not be deleted.'
         render plain: 'failure'
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://app.egghead.io/playlists/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there.

- [ ] 🏁 **Done** Ask in the comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming the `first-timers-only` issue. We're happy to keep it assigned to you as long as you need it if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance!

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it.

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below, and [reach out in our chatroom](https://publiclab.org/chat)!

### PR DESCRIPTION
**Pull request by Aman-Jamshed**
I have updated the file comment_controller.rb as suggested. Please check!

Fixes #11430  

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

File: app/controllers/comment_controller.rb [hunk 1]
```diff
@@ -45,9 +45,9 @@ def create
               "<a href='/subscribe/tag/#{tagname}'>#{tagname}</a>"
             end
             tagnames = tagnames.join(', ')
-            tagnames = " Click to subscribe to updates on these tags or topics: " + tagnames unless tagnames.empty?
++           tagnames = "Click to subscribe to updates on these tags or topics: #{tagnames}" unless tagnames.empty?
```
Comment (TildaDares) [inline app/controllers/comment_controller.rb:48]:
```suggestion
           tagnames = "Click to subscribe to updates on these tags or topics: #{tagnames}" unless tagnames.empty?        
```

**Review by TildaDares — CHANGES_REQUESTED** [review]
Hi @Aman-Jamshed, you can remove the `+` sign. Thank you!!

**Review by TildaDares — APPROVED** [review]
Amazing work @Aman-Jamshed. Thank you!!

**Comment by Aman-Jamshed** [comment]
Hey @TildaDares If everything seems good to you please merge my PR. And please let me know what should I do next?

**Comment by TildaDares** [comment]
Congratulations on merging your PR @Aman-Jamshed 🎉🎉

**Comment by Aman-Jamshed** [comment]
Thankyou so much @TildaDares for helping!
I also applied to Outreachy for the upcoming cohort, which is still pending approval. Please help me what to do next to get familiar with the codebase and to contribute more. I'd be grateful!

**Comment by TildaDares** [comment]
Hi @Aman-Jamshed, you can look at the issues tab to find issues that interest you. You can also take a look at the publiclab.org to find bugs that you can fix. Thank you!

**Pull request merged**